### ISSUE TITLE
GraphQL, gql template literals, and including fragments

### ISSUE THREAD

**Issue by baerrach** [prettier/prettier#2640]
Currently I am writing my graphql in a *.js file, where the query is referencing the fragment from a JavaScript variable that contains the fragment gql. However prettier doesn't recognise the ${} stuff and isn't reformatting the code.

Its entirely possible I'm not doing this in the recommended way (e.g. not using *.graphql files). Updates to existing documentation to put people on the right path would be appreciated.

```graphql
fragments {
  all: gql`...`,
};

query = gql`
query allPartsByManufacturerName(
  $name: String!
) {
  allParts(filter:{
    manufacturer: {
      name: $name
    }
  }) {
    ...PartAll
  }
}
${fragments.all}
```

**Comment by azz** [comment]
We don't currently support expressions inside tagged template GraphQL literals. We should add this.

**Comment by lydell** [comment]
This was fixed by #3370 :tada:
